online-shop/src/app/components/presentational/products-details-view/products-details-view.component.html

```diff
@@ -0,0 +1,47 @@
+<html>
+<head>
+    <title>Product details</title>
+</head>
+<body>
```

Comment:
Angular components aren't fully standalone pages, they're just bits and pieces of HTML. You have index.html at root level where we declare this. Please do not use html, head, body tags anywhere in the application.